util/argo/audit_logger_test.go

```diff
@@ -91,7 +91,7 @@ func TestLogAppEvent(t *testing.T) {
 	}
 
 	output := captureLogEntries(func() {
-		logger.LogAppEvent(&app, ei, "This is a test message", "")
+		logger.LogAppEvent(&app, ei, app.Spec, "This is a test message", "")
```

Comment:
The tests are failing as the new parameter is not being accepted here.

Reply:
Yep, just removed that thanks